wolfcrypt/src/asn.c

```diff
@@ -24040,6 +24056,93 @@ int ParseCertRelative(DecodedCert* cert, int type, int verify, void* cm)
     return ret;
 }
 
+int ParseCertRelative(DecodedCert* cert, int type, int verify, void* cm)
+{
+    return ParseCertRelativeEx(cert, type, verify, cm, NULL);
```

Comment:
For local functions I prefer not to have an Ex version (reduces call stack if we can just change it).